### ISSUE TITLE
[MNK] Not available

### ISSUE THREAD

**Issue by yao108** [attickdoor/XIVComboPlugin#204]
![2022-02-23_16-52-01-993_v](https://user-images.githubusercontent.com/100325023/155436123-7067cef9-72a3-4b4e-a2ca-41433835955c.png)

Not sure why, or if others are having the issue, but Monk[MNK] is not showing up in the plugin menu for me.

**Comment by CMDRNuffin** [comment]
Probably because monk doesn't have any viable combos for this plugin to support. The few mutually exclusive skills monk has already replace each other, and it has no braindead 1-2-3 combos either (because with monk, at every point in your combo you have like 3 skills to choose from).

**Comment by Cyinthia** [comment]
> Probably because monk doesn't have any viable combos for this plugin to support. The few mutually exclusive skills monk has already replace each other, and it has no braindead 1-2-3 combos either (because with monk, at every point in your combo you have like 3 skills to choose from).

Monk previously had support due to aoes basically being a 1-1-1 chain although small it was a nice QoL. I'm not sure why it was removed given the fact that none of those skills were touched during the rework.

**Comment by jesse-williams** [comment]
The main problem is Formless Fist and Perfect Balance temporarily remove the need for a rotation, you can perform any move from any form without being in that form, and Perfect Balance also grants you the benefit as if you were in the requisite form. Depending on what you're going to use Masterful Blitz for, what skills should the combo buttons display during Perfect Balance? Same goes for the AOE skills, though I have yet to see a fight last long enough requiring near continuous AOE that you would fill up your Beast/Nadi using only AOE.

Additionally, even when not using Formless/Perfect, your rotation looks like this:

Dragon > Twin > Demolish
Bootshine > True > Snap
Dragon > Twin > Snap
Bootshine > True > Demolish
Dragon > Twin > Snap
Bootshine > True > Snap

Due to the DoT duration provided by Demolish.

You could possibly argue making Howling Fist/Enlightenment show Meditation when not ready, except that function is already built into the game under Steel Peak/Forbidden Chakra doing so, and while it could be construed as a QoL benefit by not limiting your button placement, it doesn't remove the number of buttons on your hotbars which is the point of this plugin.

TL;DR

You could add combo button replacements, but not without a big caveat warning box to acknowledge that you're severely limiting your DPS and will be unable to complete the requisites for two of the four abilities available through Masterful Blitz.

**Comment by attickdoor** [comment]
an intelligent write-up. thanks. not happening.
